### DISCUSSION TITLE
[BUG] Get-PnPUserProfileProperty outputs hashtable

### DISCUSSION THREAD

**Discussion by Nick-MAC** [pnp/powershell#3567] [General]
### Reporting an Issue or Missing Feature
The command Get-PnPUserProfileProperty  now outputs as a hashtable. This prevents running a " | Select " against any of the properties that matter.

Moreover - When you run Get-PnPUserProfileProperty -Account $UPN -properties <SomePropertyHere>  Many of the properties come back blank.

### Expected behavior 
The ability to run a " | Select " against the output like you could in the past. or at minimum not return blank output when you run Get-PnPUserProfileProperty -Account $UPN -properties <SomePropertyHere>

### Actual behavior 
N/A see above

### Steps to reproduce behavior
Get-PnPUserProfileProperty -Account $UPN | Select PersonalUrl
Get-PnPUserProfileProperty -Account $UPN -properties PersonalUrl

### What is the version of the Cmdlet module you are running?
on Powershell 7.3 and PNP version 2.2

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by Nick-MAC** [comment]
I see this was converted from an issue to a discussion. Does that mean this is by design?

Does anyone from the team have some light to shed on this? This was not the behavior of this command not so long ago.

A little help would be greatly appreciated!

**Reply by AndersRask** [reply]
Yes this was changed a while back.

If the properties come back blank, that just means they are empty.

You get a dictionary back. Typically you would use it something like this:

```
> $upp = Get-PnPUserProfileProperty -Account "your@account.com"
> $upp.PersonalUrl
https://yourtenant-my.sharepoint.com/personal/user_yourtenant_onmicrosoft_com/
```

To list them all either just type
`> $upp`

Or if you want explicitly to use select (not sure of the scenario) then do this
`> $upp.GetEnumerator() | Select *`
